### ISSUE TITLE
Recurring event in CalDAV only blocks off availability on first day

### ISSUE THREAD

**Issue by syclops** [calcom/cal.com#4215]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

I use cal.com with my CalDAV calendar to let people schedule meetings with me. If I make a recurring meeting (say, a meeting on Fridays from 9-10am), the time slot is blocked on my calendar for the first day, but not the remaining days, causing people to sometimes schedule meetings with me during that slot.

### Steps to Reproduce

1. (for example) Create a recurring meeting starting on some day in the future (when you can still schedule meetings).
2. Set the meeting to recur for some number of weeks (e.g., the rest of the month).
3. Check availability on cal.com for the first day of the recurring meeting and verify that the time slot is not available.
4. Check availability on cal.com for subsequent meetings and verify that the time slot is available.

I have verified that the meeting is indeed visible on several other CalDAV clients (e.g., Thunderbird or my phone).

### Technical details

- Browser: Firefox 103.0.2 (though I do not know what browser others used to schedule meetings over my recurring one).
- CalDAV server is hosted through NextCloud 20.0.14.2.
- For the actual cal.com features, I am just using cal.com, not a self-hosted instance.

**Comment by syclops** [comment]
@alishaz-polymath I dug into this a bit more, and I think this might be a duplicate of #1588 - can you confirm?

If so, happy to close this issue and update in the other one.

**Comment by alishaz-polymath** [comment]
Hey @syclops 
Yeah, it appears to be a duplicate of #1588 issue.
